BusinessTampereTrafficMonitoring/tools/lane_configurator.py

```diff
@@ -41,6 +41,13 @@ def read_args():
 
 
 def print_json(args, lanes, polygons):
+    """
+    prints the result of intersection, camera_id, lane, vertices as a json
+    #Required arguments:
+      args: Dictionary of arguments(Dict)
+      lanes : List of lanes (List)
+      polygons: List of polygons(List)
```

Comment:
I don't think any of the above three lines are necessary, they are just repeating what the variable names already say.